src/System.Data.SqlClient/tests/ManualTests/SQL/SqlSchemaInfoTest/SqlSchemaInfoTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data.Common;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class SqlSchemaInfoTest
+    {
+        [CheckConnStrSetupFact]
+        public static void TestSqlSchemaInfo()
+        {
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+
+            using (SqlConnection connection = new SqlConnection(builder.ConnectionString))
+            {
+                connection.Open();
+
+                TestGetSchema(connection);
```

Comment:
Is it necessary for each of these Test methods to be in the same Fact? If the first one fails, we won't get results for the rest.